lib/SPIRV/SPIRVToLLVMDbgTran.cpp

```diff
@@ -1492,6 +1503,43 @@ DIFile *SPIRVToLLVMDbgTran::getFile(const SPIRVId SourceId) {
                    getStringContinued(SourceArgs[TextIdx], Source));
 }
 
+void SPIRVToLLVMDbgTran::setBuildIdentifierAndStoragePath() {
+#ifndef NDEBUG
```

Comment:
should it be ifndef or ifdef? can this code be compiled in debug and release with assertions modes?